tests/Microsoft.Data.Analysis.Tests/DataFrame.IOTests.cs

```diff
@@ -603,5 +603,75 @@ Stream GetStream(string streamData)
             Assert.Null(df[2, 2]);
             Assert.Null(df[5, 3]);
         }
+
+        [Fact]
+        public void TestWriteCsvWithHeader()
+        {
+            using MemoryStream csvStream = new MemoryStream();
+            DataFrame dataFrame = MakeDataFrameWithAllColumnTypes(10, true);
+
+            DataFrame.WriteCsv(dataFrame, csvStream);
+
+            var text = Encoding.UTF8.GetString(csvStream.ToArray());
```

Comment:
Can you try replacing the 4 lines here (and in the other test cases) with `DataFrame readIn = DataFrame.LoadCsv(csvStream)`? That would test that we are able to read in what we wrote out.

Reply:
@pgovind I'm not sure how I can test WriteCsv if I replace it with LoadCsv. Could you, please, give me more details?

Reply:
Sure, what I meant was, instead of doing `csvStream.ToArray()` and reading the stream that way, can you try with `DataFrame.LoadCsv(csvStream)` and then test the resulting DataFrame's values? I see that you have already tried this but commented out that line in other test cases( like `readIn = DataFrame.LoadCsv...`). Any reason why?